src/webfaf/problems.py

```diff
@@ -65,8 +65,8 @@ def query_problems(_, hist_table,
 
     rank_query = (db.session.query(Problem.id.label('id'),
                                    func.sum(hist_table.count).label('rank'))
-                  .join(Report)
-                  .join(hist_table))
+                  .join(Report, Report.problem_id == Problem.id)
+                  .join(hist_table, hist_table.report_id == Report.id))
```

Comment:
Just a nitpick.. you don't need to specify the `... ON ...` for the `hist_table`. 
You only need to explicitly specify the first join (because it's not clear whether `Problem.id.label('id')` should be used or the `func.sum(hist_table.count).label('rank')`).

Reply:
I’m leaning towards being (overly) explicit, even if there won’t be more columns to confuse things in the near future.